app/views/foods/index.html.erb

```diff
@@ -1,4 +1,37 @@
-<div>
-  <h1 class="font-bold text-4xl">Foods#index</h1>
-  <p>Find me in app/views/foods/index.html.erb</p>
+<div class="w-full">
+<div class="flex justify-between">
+  <h1 class="font-bold text-2xl">Food List</h1>
+  <%= link_to "Add Food", new_food_path, class:"bg-teal-500 text-white font-bold py-2 px-4 rounded" %>
+</div>
+
+  <div class="not-prose relative bg-slate-50 rounded-xl overflow-hidden dark:bg-slate-800/25  mt-4">
+  <div class="relative rounded-xl overflow-auto">
+  <div class="shadow-sm overflow-hidden my-6">
+    <table class="border-collapse table-auto w-full text-sm">
+  <thead>
+    <tr>
+      <th class="border-b font-medium pl-8 pt-0 border-teal-900 bg-slate-100"><%= link_to "Food", :sort => "food"%> </th>
+      <th class="border-b font-medium pl-8  border-teal-900 bg-slate-100"><%= link_to "Measurement unit", :sort => "measurement_unit"%> </th>
+      <th class="border-b font-medium pl-8 border-teal-900 bg-slate-100"><%= link_to "Unit Price", :sort => "avg_ranking"%> </th>
+      <th class="border-b font-medium pl-8 border-teal-900 bg-slate-100"> Actions </th>
+    </tr>
+  </thead>
+  <tbody class="bg-white dark:bg-slate-800 text-center">
+  <% @foods.each do |food| %>
+  <tr>
+    <td class="border-b border-slate-200 dark:border-slate-600 p-4 pl-8 text-slate-500 dark:text-slate-400"><%= food.name %></td>
+    <td class="border-b border-slate-200 dark:border-slate-600 p-4 pl-8 text-slate-500 dark:text-slate-400"><%=food.measurement_unit %></td>
+    <td class="border-b border-slate-200 dark:border-slate-600 p-4 pl-8 text-slate-500 dark:text-slate-400">  <%=food.price %></td>
+    <td class="border-b border-slate-200 dark:border-slate-600 p-4 pl-8 text-slate-500 dark:text-slate-400">  <%= button_to "Delete", food_path(food.id), method: :delete, class:"bg-red-500 text-white font-bold py-2 px-4 rounded" %></td>
+  </tr>
+  <% end %>
```

Comment:
Don't forget to include a link to `foods#show`.

Reply:
Not sure where would the food#show route would go, as that is not within the current parameters. Let me know if you have any ideas in mind and I can give it a go.

Reply:
Since you are looping `@foods`, you can access `food.id` on each iteration and add a link to `food_path(food.id)`. It's the same you did with the delete button but instead you can use a the `link_to` method.

You can put the link on the food name. It will look something like this.

```ruby
<%= link_to food.name, food_path(food.id) %>
```

Or you can create a new column with a link that says "Details".